articles/connections/references/apple-native/guides/add-siwa-to-native-app.md

```diff
@@ -0,0 +1,91 @@
+
+7. Create a Service ID that represents your application. This might look like a redundant effort, but Apple organizes it so you can nest multiple Services IDs under the same App ID. This makes sense when you have distinct versions of your application to support different devices.
+
+8. Click the round, blue icon next to the **Identifiers** header, then choose **Services IDs** and click  **Continue**. 
+
+9. Fill in the same fields as in step 4 above (description and identifier), and enable the **Sign In with Apple** feature. 
+
+10. Click **Configure**. Apple displays a dialog where you define the web domain you will use (`<YOUR CUSTOM DOMAIN>.com`) and add a **Return URL** (`https://<YOUR CUSTOM DOMAIN>/login/callback`). Click **Save**.  
+
+11. On the **Register a Services ID** page, click **Continue** and, on the next screen, click **Register**.
+
+    After registering your Service ID, Apple redirects you to the **Certificates, Identifiers & Profiles** page. There, you will see your newly-created Service ID.
+  
+12. Click the Service ID and view the details of the service. Click **Configure** next to the **Sign In with Apple** feature. 
+
+    This time, Apple displays two new buttons next to your domain: **Download** and **Verify**.
+
+13. Click **Download** for Apple to send you a file called `apple-developer-domain-association.txt`. You will have to use the contents of this file soon. Also, keep the page open because you will have to use the **Verify** button later.
+
+## Configure the client and the connection in Auth0
+
+Once you have the credentials you need from your Apple Developer account, you need to configure the application client and the connection settings in Auth0.
+
+1. On the Auth0 Dashboard, go to **Applications**. Click the gear to open the settings for your app. 
+
+2. At the bottom of the page, click **Show Advanced Settings** and go to the **Mobile Settings** tab. Under **Native Social Login**. enable the **Enable Sign In with Apple** toggle. 
+
+    ![Application Client Settings: Advanced Mobile Settings](/media/articles/connections/social/apple/apple-app-mobile-settings.png)
+
+3. Go to **Connections** > **Social**, and click on the **Apple** connection.
+
+4. On the **Settings** tab, complete the fields **Client ID** (Services ID), **Client Secret Signing Key**, and **Apple Team ID**. You can also fill in the **Key ID** but it is optional, as Apple will accept the key without the ID.
+
+    ![Application Connection Settings](/media/articles/connections/social/apple/apple-connection.png)
+
+5. Click **Save**.
+
+4. [Test the connection](/dashboard/guides/connections/test-connections-social).
```

Comment:
You can't test a native app from the web - won't work :(

Reply:
To be clear: if you followed the complete instructions set so far, it *would* work since you also configured a web component along with the native app.  However, I fear that app developers could be misled into thinking they were somehow testing their native app's connection here.